Microsoft.Toolkit.Uwp.Notifications/Toasts/Builder/ToastContentBuilder.Actions.cs

```diff
@@ -8,13 +8,16 @@
 
 namespace Microsoft.Toolkit.Uwp.Notifications
 {
-#if !WINRT
 #pragma warning disable SA1008
 #pragma warning disable SA1009
     /// <summary>
     /// Builder class used to create <see cref="ToastContent"/>
     /// </summary>
-    public partial class ToastContentBuilder
+    public
+#if WINRT
```

Comment:
Seems like you're using both `WINRT` and `WINDOWS_UWP`, just curious what the difference is?

Reply:
WINDOWS_UWP refers to whether the Windows 10 APIs are available (.NET Standard has that disabled, but all the rest have that enabled).

WINRT refers to whether it's compiled as a Windows Runtime Component (the format that allows C++ UWP apps to consume the APIs).